stf-test/test.ml

```diff
@@ -113,7 +151,23 @@ let stf_alco_test include_dir stf_file p4_file =
       let ic = In_channel.create stf_file in
       let lexbuf = Lexing.from_channel ic in
       let stmts = Test_parser.statements Test_lexer.token lexbuf in
-      run_test' include_dir p4_file stmts [] [] [] Eval.V1Interpreter.empty_state) in
+      let env, prog = 
+        Petr4_parse.parse_file include_dir p4_file false
+        |> (function `Ok p -> p | _ -> failwith "Petr4 parser error")
+        |> Elaborate.elab
+        |> Checker.check_program
```

Comment:
Doesn't typecheck. Maybe change to `|> fun (prog, renamer) -> Checker.check_program renamer prog`

Reply:
it typechecks for me thats strange

Reply:
Ah, this was after merging master into it locally.  The elaborator returns a program and a renamer now